assets/js/base/components/cart-checkout/totals/shipping/test/index.tsx

```diff
@@ -0,0 +1,284 @@
+/**
+ * External dependencies
+ */
+import { screen, render } from '@testing-library/react';
+import { SlotFillProvider } from '@woocommerce/blocks-checkout';
+import { previewCart as mockPreviewCart } from '@woocommerce/resource-previews';
+import * as wpData from '@wordpress/data';
+import * as baseContextHooks from '@woocommerce/base-context/hooks';
+
+/**
+ * Internal dependencies
+ */
+import { TotalsShipping } from '../index';
+
+jest.mock( '@wordpress/data', () => ( {
+	__esModule: true,
+	...jest.requireActual( '@wordpress/data' ),
+	AsyncModeProvider:
```

Comment:
I think this can be removed actually, since we are requiring `@wordpress/data` in `setup-globals.js`